services/common/src/main/java/com/linkedin/openhouse/common/Constants.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.openhouse.common;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Constants {
+  public static final String HTTPHEADER_CLIENT_NAME = "X-Client-Name";
+
+  public static final String CLIENT_NAME_DEFAULT_VALUE = "";
```

Comment:
shall we have more information default value

Reply:
the main user for this metric is prometheus formatted metric scrapers.

these scrapers would want to query with filters by either "trino" (for example) or not use the filter at all, or use aggregates like distinct or group by. in this last case (distinct or group by) so **yes, it would be more informative to have a string value of "unspecified" vs ""** for use cases like displaying "unspecified" in a linechart.

didn't see a clear best practice or required spec online, but did find this advice from a knowledgeable user of prometheus:
> Prometheus treats empty string as label not being present at all, but it's not recommended to have inconsistent sets of labels on your metrics, as it makes them harder to aggregate over.  A concrete value like "-" is better.

https://groups.google.com/g/prometheus-users/c/18UG8HFiFLg?pli=1

Reply:
fixed to "unspecified"